### PR TITLE
Storage for task list

### PR DESCRIPTION
**Pull request by angkl0**

### CODE REVIEW

File: src/main/java/seedu/address/model/util/SampleDataUtil.java [hunk 1]
```diff
@@ -40,6 +46,15 @@ public static Person[] getSamplePersons() {
         };
     }
 
+    public static Task[] getSampleTasks() {
+        return new Task[] {
```
Comment (parnikkapore) [inline src/main/java/seedu/address/model/util/SampleDataUtil.java:50]: TODO: Bring these into a shared constant

File: src/main/java/seedu/address/storage/StorageManager.java [hunk 2]
```diff
@@ -18,13 +19,16 @@ public class StorageManager implements Storage {
...
     private AddressBookStorage addressBookStorage;
+    private TaskListStorage taskListStorage;
     private UserPrefsStorage userPrefsStorage;
 
     /**
      * Creates a {@code StorageManager} with the given {@code AddressBookStorage} and {@code UserPrefStorage}.
      */
-    public StorageManager(AddressBookStorage addressBookStorage, UserPrefsStorage userPrefsStorage) {
+    public StorageManager(
+            AddressBookStorage addressBookStorage, TaskListStorage taskListStorage, UserPrefsStorage userPrefsStorage) {
```
Comment (parnikkapore) [inline src/main/java/seedu/address/storage/StorageManager.java:29]: I like the position of the line break here!

**Review by parnikkapore — CHANGES_REQUESTED** [review]
Looking great! Just one renamed parameter and it's ready to merge.

File: src/main/java/seedu/address/model/util/SampleDataUtil.java (hunk 1 above)
Comment (parnikkapore) [reply src/main/java/seedu/address/model/util/SampleDataUtil.java:50]: (To clarify, no need to do this now, this is a note for later)

File: src/main/java/seedu/address/model/ModelManager.java [hunk 3]
```diff
@@ -82,6 +82,17 @@ public void setAddressBookFilePath(Path addressBookFilePath) {
         userPrefs.setAddressBookFilePath(addressBookFilePath);
     }
 
+    @Override
+    public Path getTaskListFilePath() {
+        return userPrefs.getTaskListFilePath();
+    }
+
+    @Override
+    public void setTaskListFilePath(Path addressBookFilePath) {
```
[the remark this answers is not in the document]
Comment (angkl0) [reply src/main/java/seedu/address/model/ModelManager.java]: Fixed

**Review by parnikkapore — APPROVED** [review]
LGTM!

**Pull request merged**